python/deltalake/table.py

```diff
@@ -691,6 +691,7 @@ def compact(
         partition_filters: Optional[FilterType] = None,
         target_size: Optional[int] = None,
         max_concurrent_tasks: Optional[int] = None,
+        min_commit_interval: Optional[int] = None,
```

Comment:
Not important, but it might be nice if this accepted a timedelta as well. That way users could write out `timedelta(minutes=5)` instead of `5 * 60`, which can be a little more cryptic.

Reply:
@wjones127 Done! You can pass timedelta now and it gets converted to seconds before being passed over to Rust